src/cargo/core/compiler/custom_build.rs

```diff
@@ -644,6 +648,7 @@ impl BuildOutput {
         extra_check_cfg: bool,
         nightly_features_allowed: bool,
         targets: &[Target],
+        msrv: &Option<RustVersion>,
```

Comment:
There a reason we clone earlier to then just use `&Option<T>`, rather than just staying with `Option<&T>`?

Reply:
> The unit was moved into the first closure, so it needs to be cloned. Therefore, Option<&T> cannot be passed.